src/Analyzers/Core/Analyzers/AbstractBuiltInCodeStyleDiagnosticAnalyzer.cs

```diff
@@ -8,6 +8,10 @@
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.CodeAnalysis.Options;
 
+#if CODE_STYLE
```

Comment:
All the `#if CODE_STYLE` added in this PR are temporary until Sam's changes unifies options infrastructure in shared layer.